packages/typo3-docs-theme/assets/sass/_webfonts.scss

```diff
@@ -0,0 +1,161 @@
+}
+/* source-code-pro-700 - latin_latin-ext */
+@font-face {
+    font-display: swap;
+    font-family: 'Source Code Pro';
+    font-style: normal;
+    font-weight: 700;
+    src: url('../fonts/source-code-pro/source-code-pro-v23-latin_latin-ext-700.woff2') format('woff2'); /* Chrome 36+, Opera 23+, Firefox 39+, Safari 12+, iOS 10+ */
+}
+/* source-code-pro-700italic - latin_latin-ext */
+@font-face {
+    font-display: swap;
+    font-family: 'Source Code Pro';
+    font-style: italic;
+    font-weight: 700;
+    src: url('../fonts/source-code-pro/source-code-pro-v23-latin_latin-ext-700italic.woff2') format('woff2'); /* Chrome 36+, Opera 23+, Firefox 39+, Safari 12+, iOS 10+ */
+}
+
+/* source-sans-3-200 - latin_latin-ext */
+@font-face {
+    font-display: swap;
+    font-family: 'Source Sans 3';
+    font-style: normal;
+    font-weight: 200;
+    src: url('../fonts/source-sans-3/source-sans-3-v15-latin_latin-ext-200.woff2') format('woff2'); /* Chrome 36+, Opera 23+, Firefox 39+, Safari 12+, iOS 10+ */
+}
+/* source-sans-3-200italic - latin_latin-ext */
+@font-face {
+    font-display: swap;
+    font-family: 'Source Sans 3';
+    font-style: italic;
+    font-weight: 200;
+    src: url('../fonts/source-sans-3/source-sans-3-v15-latin_latin-ext-200italic.woff2') format('woff2'); /* Chrome 36+, Opera 23+, Firefox 39+, Safari 12+, iOS 10+ */
+}
+/* source-sans-3-300 - latin_latin-ext */
+@font-face {
+    font-display: swap;
+    font-family: 'Source Sans 3';
+    font-style: normal;
+    font-weight: 300;
+    src: url('../fonts/source-sans-3/source-sans-3-v15-latin_latin-ext-300.woff2') format('woff2'); /* Chrome 36+, Opera 23+, Firefox 39+, Safari 12+, iOS 10+ */
+}
+/* source-sans-3-300italic - latin_latin-ext */
+@font-face {
+    font-display: swap;
+    font-family: 'Source Sans 3';
+    font-style: italic;
+    font-weight: 300;
+    src: url('../fonts/source-sans-3/source-sans-3-v15-latin_latin-ext-300italic.woff2') format('woff2'); /* Chrome 36+, Opera 23+, Firefox 39+, Safari 12+, iOS 10+ */
+}
+/* source-sans-3-regular - latin_latin-ext */
+@font-face {
+    font-display: swap;
+    font-family: 'Source Sans 3';
+    font-style: normal;
+    font-weight: 400;
+    src: url('../fonts/source-sans-3/source-sans-3-v15-latin_latin-ext-regular.woff2') format('woff2'); /* Chrome 36+, Opera 23+, Firefox 39+, Safari 12+, iOS 10+ */
+}
+/* source-sans-3-italic - latin_latin-ext */
+@font-face {
+    font-display: swap;
+    font-family: 'Source Sans 3';
+    font-style: italic;
+    font-weight: 400;
+    src: url('../fonts/source-sans-3/source-sans-3-v15-latin_latin-ext-italic.woff2') format('woff2'); /* Chrome 36+, Opera 23+, Firefox 39+, Safari 12+, iOS 10+ */
+}
+/* source-sans-3-500 - latin_latin-ext */
+@font-face {
+    font-display: swap;
+    font-family: 'Source Sans 3';
+    font-style: normal;
+    font-weight: 500;
```

Comment:
So we really need 500, 600, 800 and 900?

400 is normal
700 is bold

Reply:
yeah you are right, 700 and regular with their italics should be enough. My designer in the company always wants all the weights but we dont seem to use them. Changed it